src/Pages/SignUp/SignUp.js

```diff
@@ -1,7 +1,170 @@
-import React from "react";
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+import SignUpInput from "./SignUpInput";
 
 const SignUp = () => {
-  return <div>signup</div>;
+  let [userName, setUserName] = useState("");
+  let [userEmail, setUserEmail] = useState("");
+  let [userPw, setUserPw] = useState("");
```

Comment:
```suggestion
  const [userName, setUserName] = useState("");
  const [userEmail, setUserEmail] = useState("");
  const [userPw, setUserPw] = useState("");
```
나중에 새로 할당할 값이 아니기 때문에 const를 쓰지 않을 이유가 없을거 같습니다!

Reply:
반영했습니다 👍🏻